### PR TITLE
TINY-11463: Improve editor content highlighting when using accessibility checker

### PR DESCRIPTION
**Pull request by ArvinJ-H**
Related Ticket: 
TINY-11463

Description of Changes:
- Updated CSS to reflect new a11y design

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 
 @accessibility-issue-info-heading-text-color: contrast(@dialog-background-color, @color-white, @color-tint);
 @accessibility-issue-info-body-text-color: contrast(@dialog-background-color, @color-white, @text-color);
-@accessibility-issue-info-background-color: contrast(@dialog-background-color, fade(darken(@color-tint, 3%), 10%), fade(darken(@color-tint, 3%), 40%));
+@accessibility-issue-info-background-color: mix(@background-color, @color-tint, @overlay-opacity);
 
 @accessibility-issue-warn-heading-text-color: contrast(@dialog-background-color, @color-white, darken(#cc8500, 12%));
 @accessibility-issue-warn-body-text-color: contrast(@dialog-background-color, @color-white, @text-color);
-@accessibility-issue-warn-background-color: contrast(@dialog-background-color, fade(orange, 8%), fade(orange, 50%));
+@accessibility-issue-warn-background-color: mix(@background-color, #FFCC00, @overlay-opacity);
```
Comment (kimwoodfield) [inline modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:12]: Hey Arvin, should we extract `#FFCC00` into its own variable like we did for `info` and `error`?

**Review by kimwoodfield — APPROVED** [review]

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 1 above)
Comment (ArvinJ-H) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less]: done

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less [hunk 2]
```diff
@@ -1,21 +1,22 @@
 //
 //
 // Accessibility checker
+@overlay-opacity: 80%;
```
Comment (spocke) [inline modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:4]: Shouldn't this be named `@accessibility-overlay-opacity` or something if that is specific to the a11ychecker?

**Review by spocke — APPROVED** [review]

File: modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less (hunk 2 above)
Comment (ArvinJ-H) [reply modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less:4]: yeah true, fixed

**Review by MitchC1999 — APPROVED** [review]

**Pull request merged**